README.md

```diff
@@ -1,25 +1,218 @@
+
+### **Tips :**
+If you using wide angle camera like [this](https://www.kubii.fr/cameras-capteurs/2207-module-camera-5mp-grand-angle-kubii-3272496011205.html), we have modelize on the chassis a camera cap holder. Convenient to avoid losing it
+![chassis](3dDesigns/camera-cover/preview.png)
+> 📒 [open](3dDesigns/camera-cover/preview.png) chassis stl file
+
+
+
+
+## 🧪 Create your ansible recipe
+
+*** 
+If you wanted to use our features and improvements you should create your own recipe book with your wanted recipes. Indeed, ansible has a configuration file where we specify what options ( recipe ) we want, so you have to modify the [donkeycar.yml](ansible/donkeycar.yml) file and comment recipes you don't want
+
+```yml
+ roles:
+    - role: raspi-config #rasperry pi configuration role
+      become: yes # This role need root privileges
+    - role: car-config # car configurations (ntp, hostname, ds4drv...)
+      become: yes # This role need root privileges
+    - role: shutdown-btn # shutdown button service
+      become: yes # This role need root privileges
+    - role: ps4-controller # install and configure ds4drv
+      become: no # This role no need root privileges
+    - role: openCV # Install openCV (optional)
+      become: yes # This role need root privileges
+    - role: donkeycar # Donkeycar installation 
+      become: no # This role doesn't need root privileges
+    - role: mycar # create car with configurations
+      become: no # This role no need root privileges
+    - role: IR-lap-timer # install dependencies for take charge IR counter
+      become: yes # This role need root privileges
+```
+> 📒 [ansible/donkeycar.yml](ansible/donkeycar.yml) preview
+
+* `raspi-config` : defaults installations like wifi credentials, ssh ...
+* `car-config` : car configurations (ntp, hostname, ds4drv...)
+* `shutdown-btn` : add scripts for the shutdown button ( see [documentation](doc/boutton/Boutton_poussoir.md) and [schematic](doc/schema/schema-electronique.png) )
+* `ps4-controller` : installation of ds4drv and configuration for ps4 bluetooth controllers
+* `openCV` : install openCV according to donkeycar documentation
+* `donkeycar` : install [donkeycar project](https://github.com/autorope/donkeycar) and its requirements 
+* `mycar` : create mycar folder with its files
+* `IR-lap-timer` : Infrared installation and configuration
+
+**Don't comment required recipes (raspi-config, car-config, donkeycar, mycar ), this action can prevent the proper functioning of installations or configurations**
+
+> ⚠️ Some features have hardware requirement. **Please read our [electronic schematic](doc/schema/schema-electronique.png)
+
+## ⚙️ Modify default car configurations
+
+***
+
+- modify configuration for single car
+
+  Before launching your recipe, you should modify config var with your configurations like `wpa_suplicant` to specify acces point name and credentials, donkeycar steering, throttle configuration and ip's of donkeycar servers
```

Comment:
a recipe is deployed not launched

and the ip of the donkeycar server (there's only one also that 's is useless on ip if you use "of" right after)